regulations/templates/regulations/toc-preamble.html

```diff
@@ -0,0 +1,14 @@
+<ol>
+  {% for item in preamble_toc %}
+  <li>
+    <a
+        href="{{item.url}}"
+        data-section-id="{{item.full_id}}">
+      {{item.title}}
+    </a>
+    {% with preamble_toc=item.children %}
+      {% include "regulations/toc-preamble.html" %}
```

Comment:
Heads up: when we were working with Django templates before, we found that they didn't handle recursion properly; we needed to convert template names into a variable. I don't know if this is still the case, but it'd be apparent at level three (the template is compiled to include itself, so levels 1 and 2 display properly).